pkg/vault/contracts/ProtocolFeeController.sol

```diff
@@ -149,6 +156,16 @@ contract ProtocolFeeController is
         return getVault();
     }
 
+    /// @inheritdoc IProtocolFeeController
+    function collectAggregateFees(address pool) public {
+        getVault().unlock(abi.encodeWithSelector(ProtocolFeeController.collectAggregateFeesHook.selector, pool));
+    }
+
+    function collectAggregateFeesHook(address pool) external onlyVault {
```

Comment:
```suggestion
    /**
     * @dev Copy and zero out the `aggregateFeeAmounts` collected in the Vault accounting, supplying credit
     * for each token. Then have the Vault transfer tokens to this contract, debiting each token for the amount
     * transferred so that the transaction settles when the hook returns.
     */
    function collectAggregateFeesHook(address pool) external onlyVault {
```